kuzzleio/kuzzle-admin-console
# [KZL-1441] realtime view

## What does this PR do ?
Redesigns the Realtime view for a collection.

### How should this be manually tested?
* Run the tests and ensure that _all the tests except from `Users` are green_
* play with it on Netlify

## Other changes
* Rewritten the API of the `JSONEditor` component to make it more usable and reliable
